### DISCUSSION TITLE
v5.0.0-rc.0

### DISCUSSION THREAD

**Discussion by prestonvanloon** [OffchainLabs/prysm#13630] [General]
Release candidate for v5.0.0, including:

- Mainnet deneb fork config
- Full notes to come in the final release.

Full diff https://github.com/prysmaticlabs/prysm/compare/v4.2.1...v5.0.0-rc.0

<hr /><em>This discussion was created from the release <a href='https://github.com/prysmaticlabs/prysm/releases/tag/v5.0.0-rc.0'>v5.0.0-rc.0</a>.</em>

**Comment by cromaniuc** [comment]
Any idea when v5.0.0 final will be released? I'd like to prepare the infrastructure beforehand. Thanks!

**Reply by prestonvanloon** [reply]
Tomorrow, Feb 22nd
